### ISSUE TITLE
Test mIoU of ScanNet v2

### ISSUE THREAD

**Issue by HaoRan-hash** [Matrix-ASC/DeLA#10]
I train DeLA on ScanNet v2 (train and val). Then I inference the test set of ScanNet v2 and submit result to official website. The test mIoU is 74.0. 
I am very confused. Why the val mIoU is high but the test mIoU is low ? (compared to Ptv2, etc.)
I appreciate your work and would like to know the reason for the above problem.

**Comment by binjiechen** [comment]
It’s worth noting that:
① Different methods adopt different strategies for the test set. For example, Stratified Transformer trained an ensemble of models, Point Transformer V2 uses Mix3D, OctFormer uses an initial grid size of 0.01m, Swin3D does pretraining and ensembling. Therefore the results are not directly comparable. The exact strategy is often not public (unlike val set that has reproducible code). The best strategy is also likely method specific.
② The test set has fewer samples than the validation set (100 compared to 312), and likely possesses larger variance. For example, you can check out this [PTv2 issue](https://github.com/Gofinge/PointTransformerV2/issues/34).

Honestly, I'm reluctant to put effort into this (get a high mIoU on test set), but maybe someday I will.
